infra/README.md

```diff
@@ -0,0 +1,35 @@
+# Deploying the infrastructure
+
+You can deploy the Kernel Memory infrastructure to Azure by clicking the button below. This will create required resources. We recommend to create a new resource group for this deployment.
```

Comment:
```suggestion
You can deploy the Kernel Memory infrastructure to Azure by clicking the button below. This will create required resources. We recommend to create a new resource group for each deployment.
```